centos/common/hpc-tuning.sh

```diff
@@ -14,23 +14,74 @@ cat << EOF >> /etc/security/limits.conf
 EOF
 
 # Enable reclaim mode
-echo "vm.zone_reclaim_mode = 1" >> /etc/sysctl.conf
```

Comment:
Why do we remove it?
If we remove it, should we also remove the comment above it?